Update readme and admin guide docs (#138)
mattermost/mattermost-plugin-confluence: README.md

  - Confluence spaces, including those created, updated, deleted, and restored, and those with added comments.
  - Confluence pages, including those created, updated, deleted, restored, and those with added, deleted, or updated comments.

> **Note**: For Confluence server version greater than 9, user need to be connected to mattermost in order to get the notifications or the admin has to setup API token.

Reviewer: ```suggestion
**Note**: For Confluence server version greater than 9, users need to be connected to Mattermost in order to get Confluence notifications, or the administrator is required to setup an API token.
```

This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/integrate/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/integrate/plugins/developer-setup/) for more information about developing and extending plugins.

## Note for Confluence server version greater or equal to 9

- For Confluence server version greater than or equal to 9, run `/confluence install server` and follow the setup instructions.

> - User how triggers the event on confluence need to be connected on Mattermost in order to get the notification

Reviewer: ```suggestion
- User who triggers the event on Confluence must be connected to Mattermost in order to get the notification
```

> - Generic notification will be recieved for Page subscriptions if the user who is not connected to Mattermost triggers the event on Confluence

Reviewer: ```suggestion
- Generic notifications will be received for Page subscriptions if the user who triggers the event on Confluence is not connected to Mattermost 
```

> - Admin can setup Admin API Token in plugin configuration to allow notifications for events even when the user who triggers the event on confluence is not connnected on Mattermost

Reviewer: ```suggestion
- Administrators can setup an Admin API Token in the plugin configuration to allow notifications for events even when the user who triggers the event on Confluence is not connected to Mattermost
```

### / confluence connect

> Connect your mattermost account to confluence.

Reviewer: ```suggestion
Connect your Mattermost account to Confluence.
```

### / confluence disconnect

> Disconnect your mattermost account to confluence.

Reviewer: ```suggestion
Disconnect your Mattermost account to Confluence.
```
